src/main/kotlin/com/openlattice/linking/controllers/RealtimeLinkingController.kt

```diff
@@ -52,6 +51,23 @@ class RealtimeLinkingController(
         return linkableEntitySets.minus(entitySetsNeedLinking)
     }
 
+    @RequestMapping(
+            path = [RealtimeLinkingApi.MISSING],
+            method = [RequestMethod.POST],
+            produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun getEntitiesMissingLinking(@RequestBody entitySetIds: Set<UUID>): Map<UUID, Set<UUID>> {
+        ensureAdminAccess()
+        val linkableEntitySets = lqs
+                .getLinkableEntitySets(linkableTypes, entitySetBlacklist, whitelist.orElse(setOf()))
+                .toSet()
+        val entitiesNeedLinking = HashMap<UUID, Set<UUID>>()
```

Comment:
Why not do a batch call? You load those UUIDs anyway into memory

Reply:
`lqs.getEntitiesNotLinked(linkableEntitySets)
    .groupBy { it.first }
    .mapValues { it.value.map { it.second }.toSet() }`

Reply:
^ this will map it to the desired output type: First you group the pairs by entity set id (`it.first`) , which results in `Map<UUID, List<Pair<UUID, UUID>>>` and then you map this maps values